### ISSUE TITLE
Add collaborators?

### ISSUE THREAD

**Issue by maenu** [maenu/grunt-template-jasmine-istanbul#38]
@gyandeeps asked in #37 what I think about adding collaborators to this project. I thought about this earlier but there was no one else asking for it, so I didn't spend too much time on this thought. @gyandeeps and others' recent activity has shown that there are actually people relying on this project in productive environments which I think is cool, but brings a bit of responsibility.

I currently do not have the resources to do much more than fixing bugs (well, actually I am just merging your pull requests, so thanks for that ;)). Since I am currently also not actively developing with JavaScript, I am currently not using this project and therefore do not see how I would be able to provide more resources for it in the near future. So currently this project has a maintenance problem. I certainly don't want this project to die since it still feels like my baby and it makes me quite proud to have contributed something to the open source community that is actually used.

Since this is the first project of me that gathered this kind of attention, I do not know how to handle such a situation. A bunch of questions are going through my head to which if have no final answers:

**How do I manage collaboration with people I do not know?**
I want to know the collaborators to some extent. Who are they, where do they come from, what are they working on, what is their motivation? On which constraints do I select collaborators?

**What happens when the collaborators and I do not agree on a specific issue?**
I want to stay some kind of gate keeper, since I think this is the easiest way to keep quality in all aspects at a constant hight level. My fear is that collaborators might want to change too much which might lead to a second system effect. On the other hand, a gate-keeper is a bottle-neck and might not improve the situation in an effective way. I think we need to define a clear scope of what this project includes and what it does not. Additionally we need a guideline on how collaboration works, e.g. bugs are fixed by adding tests and making sure every other test still passes. I could imagine myself in a role where bugs are fixed and new patch versions released by collaborators autonomously, but I have a veto right for enhancements and new features.

Now I would like to hear from you what you think about adding collaborators. Am I just a dictator who is afraid of changing the status quo because he could lose his position? Is there no need for collaborators? Do you have any advice/links on how to manage collaboration in an open source project? What are possible collaboration models, what are their characteristics, trade-offs and experiences? And last but not least: Who would want to be a collaborator?

**Comment by gyandeeps** [comment]
@maenu I would like to be a collaborator on this project. I work for Cerner as a Software Engineer and do a lot of JavaScript in my profile. I also do `NodeJs` and `JavaScript` on the side as I am passionate about it. I do contribute to various libraries like `eslint`, `socket.io`, etc. 

Here is a link to `eslint` governance model: http://eslint.org/docs/developer-guide/governance.html

Do let me know what do you think about it.

**Comment by gyandeeps** [comment]
Any thoughts @maenu ?

**Comment by maenu** [comment]
Hi @gyandeeps, I added you as a collaborator for this and the example repository.
You showed the enthusiasm, endurance and determination for this project, so you are definitely the better maintainer of it at the moment ;).
I also added an evolution document (see #43) where I noted some goals and methods which are important for me.
Let me know what you think about, especially if there is something you do not agree with or understand.
